### DISCUSSION TITLE
Is is possible to disable certain UI Elements?

### DISCUSSION THREAD

**Discussion by GUImodder** [smplayer-dev/smplayer#565] [Q&A]
Hi guys,

I'm aware that my use case is pretty particular but maybe you can help me out: I travel a lot and a have to download most of the content i watch. Sometimes I watch sporting events and in that case some usually great features become problematic: I need to hide every mention of the file length or the progress of the video file (for spoiler reasons).

Please look at the marked elements in the screenshots. Is there a way to disable these?
![SMplayer 1](https://user-images.githubusercontent.com/15251611/197586700-cb6f23ff-6e3b-4bc4-a32e-91813
<img width="1512" alt="Screenshot 2022-10-24 at 19 09 14" src="https://user-images.githubusercontent.com/15251611/197586720-40997856-88f0-415b-bacf-c8b71d6a5e50.png">
4d77de8.png)
![SMplayer 1](https://user-images.githubusercontent.com/15251611/197586874-16e8510b-55c3-43e4-9c40-785ea3d6de26.png)

**Comment by smplayer-dev** [comment]
You can remove the seeking bar from the OSD selecting "Subtitles only" from the menu View -> OSD.
In order to remove the time labels from the window, maybe the easiest is to change to the mini GUI in preferences -> interface. This GUI doesn't have a status bar. Still the time is displayed in full screen, to remove it, right click on an empty place in the menu bar or control bar and select "edit floating control". There remove (move to the left panel) the display time label.

**Reply by GUImodder** [reply]
Thank you very much! You have solved all my issues!
